common/rpc/interceptor/telemetry.go

```diff
@@ -271,6 +274,26 @@ func (ti *TelemetryInterceptor) emitActionMetric(
 			metrics.ActionCounter.With(metricsHandler).Record(1, metrics.ActionType("command_BatchMarkers"))
 		}
 
+		var updates int64
+		for _, msg := range completedRequest.Messages {
+			if msg == nil || msg.Body == nil {
+				continue
+			}
+			body, err := msg.Body.UnmarshalNew()
```

Comment:
It might! But I'm seeing `type.googleapis.com/temporal.api.update.v1.Acceptance` come back from `msg.Body.GetTypeUrl()` and `temporal.api.update.v1.Acceptance` from `FullName()`. I'm trying to figure out if this is a mistake in my test or just what it is.

Reply:
Okay, seems to work now.